src/config/config.py

```diff
@@ -42,12 +42,25 @@ class DBUpstreamCfg:
     enabled: bool
 
 
+@dataclass
+class ServiceTypeCfg:
+    type: str
+    image: str
+    ingress: Optional[bool] = None
+    empty_env: Optional[str] = None
+    envvars: Optional[dict[str, str]] = field(default_factory=dict)
+    ports: Optional[dict[str, str]] = field(default_factory=dict)
+    properties: Optional[dict[str, str]] = field(default_factory=dict)
+    subject_alternative_name: Optional[str] = None
+
+
 @dataclass
 class ServiceCfg:
     type: str
     tag: str
     image: str
     ingress: Optional[bool] = None
+    empty_env: Optional[str] = None
```

Comment:
what's the meaning of this empty_env of type string?

Reply:
this is something could be useful when performing database sync operations (to speed up the process), this is retained by the old implementation.